x-pack/filebeat/input/awss3/sqs_s3_event.go

```diff
@@ -133,6 +134,19 @@ func (p *sqsS3EventProcessor) ProcessSQS(ctx context.Context, msg *types.Message
 	keepaliveWg.Add(1)
 	go p.keepalive(keepaliveCtx, log, &keepaliveWg, msg)
 
+	rawRecieveCount, hasReceiveCountAttribute := msg.Attributes[sqsApproximateReceiveCountAttribute]
+
+	if hasReceiveCountAttribute {
+		if receiveCount, err := strconv.Atoi(rawRecieveCount); err == nil && receiveCount == 1 {
```

Comment:
I think a comment stating that this only contributes to the histogram on the first time the message is received to avoid skewing the metric when processing retries.